src/main/java/org/candlepin/subscriptions/subscription/jmx/SubscriptionJmxException.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021 Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package org.candlepin.subscriptions.subscription.jmx;
+
+/**
+ * JMX Bean catch-all for the subscription jmx bean's exceptions.
+ */
+public class SubscriptionJmxException extends Exception {
```

Comment:
Not for this PR, but a general thought: we should figure out a way to do this more generally (either a single JmxException class) or figure out a better way to accomplish our goal. (We were using a checked exception to ensure that we simply followed a pattern of logging the error before returning to the client, as otherwise we don't have a record of the exception in our logs). And it definitely hasn't fulfilled the intended purpose, looking at other Jmx beans, we're pretty inconsistent :-/.